apps/wallets/smart-wallet/react/src/app/auth/FirebaseAuthManager.ts

```diff
@@ -59,7 +59,7 @@ export const checkAuthState = (): Promise<string | undefined> => {
         onAuthStateChanged(auth, async (user) => {
             try {
                 if (user) {
-                    const jwt = await user.getIdToken(true);
```

Comment:
This `true` is an optional property that forces a refreshed jwt every call. We don't need that!